docs/challenges/language-support/python.mdx

```diff
@@ -0,0 +1,75 @@
+---
+title: "Python"
+description: "Learn how to solve CodeCrafters challenges in Python"
+---
+
+<Note>
+  **New to Python?** We recommend checking out [Automate the Boring Stuff with
+  Python](https://automatetheboringstuff.com/) to get up to speed.
+</Note>
+
+# Local development
+
+To run the code for your challenge locally, you'll need `Python` installed. Our test runners use `Python` version `3.12` (as of April 2024).
```

Comment:
```suggestion
To run the code for your challenge locally, you'll need `python3` installed. Our test runners use version `3.12` (as of April 2024).
```

(The value here corresponds to what CLI command they need to have - you'll see that the Java article mentions `mvn` too)